homeassistant/components/zwave_js/device_condition.py

```diff
@@ -140,17 +141,18 @@ async def async_get_conditions(
     conditions.append({**base_condition, CONF_TYPE: NODE_STATUS_TYPE})
 
     # Config parameter conditions
-    conditions.extend(
-        [
-            {
-                **base_condition,
-                CONF_VALUE_ID: config_value.value_id,
-                CONF_TYPE: CONFIG_PARAMETER_TYPE,
-                CONF_SUBTYPE: f"{config_value.value_id} ({config_value.property_name})",
-            }
-            for config_value in node.get_configuration_values().values()
-        ]
-    )
+    for config_value in node.get_configuration_values().values():
```

Comment:
`config_value` is overridden by `config_value` from the inner list comprehension. If that was the intention, I would suggest renaming it to `_` here just to avoid confusion.

Reply:
thanks for catching this!

Reply:
https://github.com/home-assistant/core/pull/66626